### PR TITLE
Channel 7 News Script

### PR DESCRIPTION
**Pull request by tkcjesse**
For the longest time there was just a random map marker up on the hill.
No npc. Nothing. Empty. Now... it has a purpose in life. <- ha get it!?

Screenshot: http://images.akamai.steamusercontent.com/ugc/277348411050855805/A907D0E84C6BF7C1ADB1EC97CF59D7010CBB4094/

### CODE REVIEW

**Comment by Jawshy** [comment]
You will also need to include updates to the BattlEye filters, specifically scripts, remoteExec, and publicVariable. 

The casing is also incorrect. [Example](https://github.com/ArmaLife/Framework/pull/773/files#diff-c01c0ab5e776c7f4d60bcd44e57b54faR17): `disableSerialization` not `disableserialization`.

**Comment by tkcjesse** [comment]
Blame BI....https://i.gyazo.com/4eea40cd8ddf4f0b118712984aefd4e8.png

**Comment by danielstuart14** [comment]
Can you add an option for enable/disable it?

**Comment by Jawshy** [comment]
> & not to be that guy, however I really cannot be bothered spending time on battlEye filters... If it wasn't all in regex I'd possibly consider it since I could just plug it all into: http://gp.gamersinc.net/BEF.php

That's funny because scripts.txt doesn't use, nor support, regex. 

Anyway, if I update the filters and give you the files, can you commit them to your pull, @tkcjesse?

File: Altis_Life.Altis/core/functions/fn_AAN.sqf [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+disableSerialization;
+
+_header = _this select 0;
+_line = _this select 1;
+_sender = _this select 2;
+
+30 cutRsc ["rscAAN","plain"];
+_display = uiNamespace getVariable "BIS_AAN";
+_textHeader = _display displayCtrl 3001;
+_textHeader ctrlSetStructuredText parseText format ["<t size='2'>%1</t><br />Broadcasted by: %2",_header,_sender];
```
Comment (setoy) [inline Altis_Life.Altis/core/functions/fn_AAN.sqf]: "Broadcasted by: " is not localized

**Comment by tkcjesse** [comment]
@Jawshy, yeah. scripts.txt, however as you even mentioned yourself there is 2 more remoteExec, and publicVariable. Which use and support regex. Also, not sure if you recall, however when BEfilters were originally added it was kinda mentioned that they'd be there if you were the one to upkeep them because "others" couldn't be bothered with it...

https://i.gyazo.com/075cb67be0a19f4f11f8efe278094bb9.png

File: Altis_Life.Altis/mission.sqm [hunk 2]
```diff
@@ -38249,6 +38248,22 @@ class Mission
...
+					dataType="Object";
+					class PositionInfo
+					{
+						position[]={4543.9028,263.88174,15401.926};
+					};
+					side="Civilian";
+					flags=5;
+					class Attributes
+					{
+						init="this enableSimulation false; this allowDamage false; this addvest ""V_Press_F""; this addheadgear ""H_Cap_press""; this adduniform ""U_NikosAgedBody""; this addAction[""<t color='#ADFF2F'>ATM</t>"",life_fnc_atmMenu,"""",1.5,false,false,"""",' isNull objectParent player && player distance _target < 4']; this addAction[localize""STR_News_AddAction"",life_fnc_newsBroadcast,"""",1.5,false,false,"""",'isNull objectParent player && player distance _target < 4'];";
```
Comment (Jawshy) [inline Altis_Life.Altis/mission.sqm]: `addVest`* `addHeadgear`* `addUniform`*

**Comment by Jawshy** [comment]
Just going to post some of my suggestions here that @tkcjesse and I also discussed on Gitter. 

> Also, could the Config_Master option hide the NPC and map marker?

> [...] a player can make the header so long that it clips off of the screen. Can a header field character limit be added [of about 72 characters]? 

[PR 773 BEFilters.zip](https://files.gitter.im/Jawshy/AUKr/PR-773-BEFilters.zip) - Updated BEFilters.

**Comment by Jawshy** [comment]
```
23:23:29 "Variable: life_fnc_postnewsbroadcast_meta is not allowed TYPE: CODE NS: MN"
23:23:29 "Variable: life_fnc_newsbroadcast_meta is not allowed TYPE: CODE NS: MN"
23:23:29 "Variable: life_fnc_newsbroadcast is not allowed TYPE: CODE NS: MN"
23:23:29 "Variable: life_fnc_postnewsbroadcast is not allowed TYPE: CODE NS: MN"
23:23:29 "Variable: life_fnc_aan is not allowed TYPE: CODE NS: MN"
23:23:29 "Variable: life_fnc_aan_meta is not allowed TYPE: CODE NS: MN"
```

**Comment by Heavybob** [comment]
that is so coool!

**Comment by danielstuart14** [comment]
okay, fix spyglass and add an option for disable it (In player Menu).  (Each player could disable it going to Y > settings)

**Comment by tkcjesse** [comment]
@danielstuart14 - I can merge it when I add more to it. I'm going to revise it later tonight. Unless you're needing it sooner?

**Comment by Jawshy** [comment]
> So you want players to disable the ability to see the broadcast on a player level?

Similar to how each player can disable Side Channel for themselves.

**Comment by tkcjesse** [comment]
Yeah I understood it. I'll also add the ability to hide the NPC I guess. I'll return to this in a bit...

**Pull request closed without merging**